packages/core/src/domain/configuration/tags.ts

```diff
@@ -43,3 +41,17 @@ export function buildTag(key: string, rawValue: string) {
 
   return `${key}:${sanitizedValue}`
 }
+
+function hasForbiddenCharacters(rawValue: string) {
+  // Unicode property escapes is not supported in all browsers, so we use a try/catch.
+  // Todo: Remove the try/catch when dropping IE11.
+  try {
+    // We use the Unicode property escapes to match any character that is a letter including other languages like Chinese, Japanese, etc.
+    // p{Ll} matches a lowercase letter.
+    // p{Lo} matches a letter that is neither uppercase nor lowercase (ex: Japanese characters).
+    // See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Unicode_character_class_escape#unicode_property_escapes_vs._character_classes
+    return new RegExp('[^(\\p{Ll}|\\p{Lo})0-9_:./-]', 'u').test(rawValue)
```

Comment:
**💬 suggestion:** ‏`(`, `|` and `)` characters should not be included in the character list. Remove them?